rendering_resource_manager_service/session/management/session_manager_settings.py

```diff
@@ -56,7 +56,7 @@
                       '/var/log/vws'
 SLURM_ERR_FILE = 'err.log'
 SLURM_OUT_FILE = 'out.log'
-SLURM_ALLOCATION_TIMEOUT = 20
+SLURM_ALLOCATION_TIMEOUT = 10
```

Comment:
sure that's not too short? I'm also not super sure, but it seems to me that's the minimum it will wait, even if it would find a node earlier. No timeout at all seems dangerous as well though...